ignite/pkg/cosmosgen/generate_openapi.go

```diff
@@ -90,6 +121,18 @@ func generateOpenAPISpec(g *generator) error {
 		}
 	}
 
+	if !anyChanges {
```

Comment:
I am also keeping track of whether there were any changes at all, because the last combine step is pretty expensive. Best to avoid it if nothing has changed.